packages/wonder-blocks-form/src/components/checkbox-core.tsx

```diff
@@ -6,21 +6,57 @@ import {addStyle} from "@khanacademy/wonder-blocks-core";
 import Icon from "@khanacademy/wonder-blocks-icon";
 
 import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
-
 import type {ChoiceCoreProps} from "../util/types";
 
+// Checkbox is in indeterminate state when `checked` is `null` | `undefined`
+export type Checked = boolean | null | undefined;
```

Comment:
**suggestion:** Move this to `/util/types.ts` so you can reference this type everywhere.